components/hab/src/command/pkg/upload.rs

```diff
@@ -52,6 +52,7 @@ pub fn start(ui: &mut UI,
              token: &str,
              archive_path: &Path,
              force_upload: bool,
+             disable_build_group: Option<&str>,
```

Comment:
Getting into the nitpicky weeds.... `disable_build` naming would be shorter and simpler